mock-data/CosmosDbSeeder/items/index_data/attribute_definitions.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": "attributes",
+  "docType": "system",
+  "attributeDefinitions": [
+    {
+      "id": "accessibility/accessible_toilet",
+      "displayName": "Accessible toilet"
+    },
+    {
+      "id": "accessibility/braille_translation_service",
+      "displayName": "Braille translation service"
+    },
+    {
+      "id": "accessibility/disabled_car_parking_on_site",
+      "displayName": "Disabled car parking on site"
+    },
+    {
+      "id": "accessibility/car_parking_on_site",
+      "displayName": "Car parking on site"
+    },
```

Comment:
As mentioned on [Slack](https://nhsuk.slack.com/archives/C062RBW4NF4/p1727106196015669), we should remove "on site" from these